BaseScript/fabfile.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import with_statement
```

Comment:
Should this be named fabfile.py? I guess we will be using some methods only, which shall be called from the UI. Why not rename it something like "initial_utility" ?